### PR TITLE
Fix urls

### PR DESCRIPTION
**Pull request by madhulika95b**
@ldko @somexpert 

I'm replacing `path` with `re_path` and using older regex because, the `<slug>` and  `<str>` being used here are interpreted as url parameters and raising Internal errors as `browse_json()` doesn't expect them.

### CODE REVIEW

File: nomination/urls.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     path("help/", nomination_help, name='nomination_help'),
     path("<slug>/lookup/", url_lookup, name='url_lookup'),
     path("<slug>/search.json", search_json, name='search_json'),
-    path("<slug>/browse/<str>/browse.json", browse_json, name='browse_json'),
+    re_path(r"^([^/]+)/browse/([^/]+)/browse.json$", browse_json, name='browse_json'),
```
Comment (ldko) [inline nomination/urls.py:17]: Should it have been called "attribute", not "str"?

**Review by ldko — COMMENTED** [review]
Is `slug` causing a problem, or just `str`?

File: nomination/urls.py (hunk 1 above)
Comment (madhulika95b) [reply nomination/urls.py:17]: I have used <slug> and <str> as path converters but they are being taken as keyword arguments. 
I just tried and checked, using - 
`path("<str:slug>/browse/<str:attribute>/browse.json", browse_json, name='browse_json'),` works fine too. 

I can use this to make sure we are replacing as many regexes in urls as possible.

**Review by ldko — APPROVED** [review]
Looks right to me :+1:

**Review by somexpert — APPROVED** [review]
Yeah this looks good to me, and also I now more fully understand those url converters :smile: :thumbsup:

**Pull request merged**